docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -49,27 +49,28 @@ The alias extension is under active development and new versions are released re
 az extension update --name alias
 ```
 
-## Alias commands file format
+## az alias
 
-Alias command definitions are written into a configuration file, located at `$AZURE_USER_CONFIG/alias`. The default value of `AZURE_USER_CONFIG` is `$HOME/.azure` on macOS and Linux, and `%USERPROFILE%\.azure` on Windows. The alias configuration file is written in the INI configuration file format. The general format for alias commands is:
+Alias provides convenient and familiar commands to manage your aliases.
 
-```
-[command_name]
-command = invoked_commands
+## az alias create
```

Comment:
Remove entire section. The process of creating an alias should be described in plain language, as non-reference material, detailing the usage of the command's `--name` and `--command` parameters.